app/models/state_file_az_intake.rb

```diff
@@ -185,11 +193,6 @@ def disqualified_from_excise_credit_fyst?
     all_filers_incarcerated || whole_credit_already_claimed || direct_file_data.claimed_as_dependent?
   end
 
-  def filing_status
-    return :head_of_household if direct_file_data&.filing_status == 5 # Treat qualifying_widow as hoh
-    super
-  end
```

Comment:
since we override the `filing_status` of an intake here for a `qualifying_widow` to become `head_of_household`, by leaving the `filing_status` as-is (same as what's in the `direct_file_data`) may be problematic & we need to ensure we update all downstream places where we reference `filing_status` (even controllers/forms)